roles/argocd/templates/values/00-main.j2

```diff
@@ -14,7 +14,16 @@ config:
   rbac:
     policy.csv: |
       p, role:admin, *, *, */*, allow
-      p, role:nada, *, *, */*, deny
+      p, role:nada, applicationsets, *, */*, deny
+      p, role:nada, applications, *, */*, deny
+      p, role:nada, logs, *, */*, deny
+      p, role:nada, exec, *, */*, deny
+      p, role:nada, certificates, *, *, deny
+      p, role:nada, accounts, *, *, deny
+      p, role:nada, gpgkeys, *, *, deny
+      p, role:nada, projects, *, *, deny
+      p, role:nada, clusters, *, *, deny
+      p, role:nada, repositories, get, *, allow
```

Comment:
Je serais même tenté de dire que cette ligne `p, role:nada, repositories, get, *, allow` qui donne l'accès en get aux dépôts de code devrait être positionnée sur l'AppProject plutôt que sur les RBAC globaux (pour être scopé sur le projet de chaque utilisateur et ne pas leur faire descendre par inadvertance le contenu des dépôts des autres utilisateurs)

*cf. https://argo-cd.readthedocs.io/en/stable/user-guide/projects/#project-scoped-repositories-and-clusters*

@ArnaudTA @Baboulinet-33 Qu'est ce que vous en pensez ?

Reply:
j'ai testé et visiblement le repository n'est pas dépendant du projet donc ce n'est pas pris en compte